src/Program.cs

```diff
@@ -48,8 +53,38 @@ private static async Task<bool> DecideRedirection()
         return isRedirect;
     }
 
-    private static void OnActivated(object? sender, Microsoft.Windows.AppLifecycle.AppActivationArguments e)
+    // Currently DevHome supports one set of command line arguments, most useful when debugging different apps within the DevHome package.
+    //
+    // For example:
+    //    --utilitylaunch DevHome.PI.Exe --utilityLaunchArgs "--application problemapp2"
+    //
+    // --utilityLaunch is the name of the utility to launch
+    // --utilityLaunchArgs are the arguments to pass to the utility. This is optional, but be sure to include the quotes if you have spaces in the arguments.
+    private static void ParseCommandLine(string[] args)
     {
-        _app?.ShowMainWindow();
+        var builder = new ConfigurationBuilder();
+        builder.AddCommandLine(args);
+        var config = builder.Build();
+
+        var utilityToLaunch = config["utilitylaunch"];
+        var utilityLaunchArgs = config["utilitylaunchargs"];
+
+        if (!string.IsNullOrEmpty(utilityToLaunch))
+        {
+            try
+            {
+                var processStartInfo = new ProcessStartInfo
+                {
+                    FileName = utilityToLaunch,
+                    Arguments = utilityLaunchArgs,
+                };
+
+                Process.Start(processStartInfo);
+            }
+            catch (Exception ex)
+            {
+                _log.Error($"Error launching utility: {ex.Message}");

```

Comment:
Should we log the exception itself?
```suggestion
                _log.Error(ex, $"Error launching utility: {ex.Message}");
```